e2e-tests/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-aleph_client = { git = "https://github.com/Cardinal-Cryptography/aleph-node/", rev = "d52ede9" }
+aleph_client = { git = "https://github.com/Cardinal-Cryptography/aleph-node/", rev = "c3c6d8b37d586b6b050d1e40675dd6b4ab6adcdb" }
```

Comment:
just a heads up. You probably want to cherry-pick changes to aleph-client into release-13. Othwewise relayer build upon this pr wont be working before the next testnet release.

Reply:
Will update once https://github.com/Cardinal-Cryptography/aleph-node/pull/1704 is merged